chart/kubeapps/templates/apprepository/rbac.yaml

```diff
@@ -43,6 +44,23 @@ rules:
       - jobs
     verbs:
       - create
+---
+# ClusterRole for managing AppRepository objects in every namespace,
+# so that we can update the finalizers on AppRepository objects
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+kind: ClusterRole
+metadata:
+  name: {{ template "kubeapps.apprepository.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
Ooops, I thought I had actually removed them. They don't have any effect, as you pointed out.
Ah, I know what happened... I copied and pasted the metadata so that all the objects had the same... and I forgot to remove the `namespace` again! Thanks for catching it!